src/System.Collections/src/System/Collections/Generic/HashSetEqualityComparer.cs

```diff
@@ -42,9 +42,9 @@ public int GetHashCode(HashSet<T> obj)
         }
 
         // Equals method for the comparer itself. 
-        public override bool Equals(object obj)
+        public override bool Equals(object? obj)
```

Comment:
above in GetHashCode, should `obj` be nullable?

Reply:
same question for `Equals(x, y)` (x, y nullable)

Reply:
Yes.  Good catches.